typescript/commands/build.py

```diff
@@ -38,3 +40,4 @@ def compile_inferred_project(self, file_name, params=""):
             "cmd": cmd,
             "file_regex": "^(.+?)\\((\\d+),(\\d+)\\): (.+)$"
         })
+        self.build_parameters = params
```

Comment:
What happens if you don't give any parameter, will the ``params`` be ``None`` or empty string?